dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -29,9 +29,8 @@ public class ObjectCreationNode extends Node {
     /** The enclosing expression of the object creation or null. */
     protected final @Nullable Node enclosingExpression;
 
-    // TODO: See issue 376
-    /** The constructor node of the object creation. */
-    protected final Node constructor;
+    /** The identifier node of the object creation. */
```

Comment:
Document what subclasses this can refer to, if there are multiple. If there is only one, improve the type.

Reply:
Thanks!